.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -108,14 +120,26 @@ jobs:
         env:
           INPUTS_TEST_FILTER: ${{ inputs.test_filter }}
           SEED: ${{ secrets.SEED_QAA_B2C }}
+          XRAY: ${{ inputs.export_to_xray }}
+          TEST_EXECUTION: ${{ inputs.test_execution }}
+          PROJECT_KEY: B2CQA
 
       - name: Upload Allure Results
         if: ${{ !cancelled() }}
         uses: actions/upload-artifact@v4.3.0
         with:
+          retention-days: 1
           name: allure-results-${{ matrix.shardIndex }}
           path: "apps/ledger-live-desktop/allure-results"
 
+      - name: Upload Xray Results
+        if: ${{ !cancelled() && inputs.export_to_xray }}
+        uses: actions/upload-artifact@v4.3.0
+        with:
+          retention-days: 1
+          name: xray-reports-${{ matrix.shardIndex }}
+          path: apps/ledger-live-desktop/tests/artifacts/xray/xray-report.json
```

Comment:
shouldn't path point to just a folder instead of file?

Reply:
The path is where I save the artifact at the end of the test execution with my Json reporter.